eventuate-log-cassandra/src/main/scala/com/rbmhtechnology/eventuate/log/cassandra/CassandraEventLog.scala

```diff
@@ -136,7 +137,7 @@ class CassandraEventLog(id: String, aggregateIndexing: Boolean) extends EventLog
 
   override def writeEventLogClockSnapshot(clock: EventLogClock): Future[Unit] = {
     import context.dispatcher
-    indexStore.writeEventLogClockSnapshotAsync(clock).map(_ => ())
+    updateIndex(clock).flatMap(_ => indexStore.writeEventLogClockSnapshotAsync(clock)).map(_ => ())
```

Comment:
If `aggregateIndexing` is set to false in `updateIndex`, `indexStore.writeEventLogClockSnapshotAsync` will be invoked twice in succession with the same `clock` parameter.
I guess this won't happen and if so will not have any considerable performance implications.